src/Textual/encodings/textpreprocessing.jl

```diff
@@ -0,0 +1,124 @@
+"""
+    TextEncoding() <: Encoding
+
+Encodes `Paragraph`s by applying various textual transforms.
+
+
+Encodes
+- `Paragraph` -> `Paragraph`
+
+"""
+struct Sanitize <: Encoding
+    tfms
+end
+
+function Sanitize()
+    base_tfms = [
```

Comment:
let's make `base_tfms` a constant in the module scope, i.e. `const DEFAULT_SANITIZERS`.

Then this definition becomes `Sanitize() = Sanitize(DEFAULT_SANITIZERS)`